src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java

```diff
@@ -6,13 +6,8 @@
 public enum CaseRole {
     CREATOR,
     APPLICANTSOLICITORONE,
-    APPLICANTSOLICITORTWO,
     RESPONDENTSOLICITORONE,
-    RESPONDENTSOLICITORTWO,
-    APPLICANTSOLICITORONESPEC,
-    APPLICANTSOLICITORTWOSPEC,
-    RESPONDENTSOLICITORONESPEC,
-    RESPONDENTSOLICITORTWOSPEC;
+    RESPONDENTSOLICITORTWO;
```

Comment:
This should be reverted for now, it will be a cleanup activity for after the release of Access Profiles